service/src/test/java/com/ge/predix/acs/audit/PredixEventProcessorTest.java

```diff
@@ -0,0 +1,45 @@
+package com.ge.predix.acs.audit;
+
+import static org.mockito.Matchers.anyObject;
+
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.MockitoAnnotations;
+import org.testng.Assert;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+import com.ge.predix.audit.sdk.AuditClient;
+import com.ge.predix.audit.sdk.exception.AuditException;
+import com.ge.predix.audit.sdk.message.AuditEventV2;
+import com.ge.predix.eventhub.EventHubClientException;
```

Comment:
To allow this import to be resolved, we probably want to explicitly add the following to the `predix` Maven profile of `acs/service/pom.xml`:

```
                <dependency>
                    <groupId>com.ge.predix.eventhub</groupId>
                    <artifactId>predix-event-hub-sdk</artifactId>
                    <version>${predix-event-hub-sdk.version}</version>
                </dependency>
```